perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -65,9 +66,17 @@ sub example_places {
 sub munge_report_new_bodies {
     my ($self, $bodies) = @_;
 
-    %$bodies = map { $_->id => $_ } grep { $_->name eq 'Thamesmead' } values %$bodies;
+    for (FixMyStreet::Cobrand::Bexley->new->body, FixMyStreet::Cobrand::Greenwich->new->body) {
+        if ($_ && $_->{id}) {
+            $bodies->{$_->id} = $_;
+        };
+    }
```

Comment:
This for loop is doing nothing, because `$_->{id}` never exists (it's `$_->id` as in the next line). So %bodies is left alone (which I guess is probably the desired outcome, as checks like this loop would already have happened in fetching the bodies they'd have to exist, I guess TfL are the only potential to consider, but presumably good to have them too? If it's working :) ).

Reply:
https://github.com/mysociety/fixmystreet/pull/4733